tool/ci/Makefile

```diff
@@ -0,0 +1,15 @@
+
+# Runs subcommand assign-reviewers
+assign: 
```

Comment:
These targets should be marked `PHONY`: https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html